test/components/linearIndeterminate.test.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import LinearIndeterminate from '../../src/components/LinearIndeterminate';
+
+describe('<LinearIndeterminate />', () => {
+  it('renders correctly', () => {
+    const tree = renderer
+      .create(
+        <LinearIndeterminate
+          classes={{
+            root: 'X',
+          }}
+        />
+      )
+      .toJSON();
+
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
Please remove this test, and the `LinearIndeterminate` code it is testing.  It is not used anywhere.

Reply:
Yes sure. I will remove the test. Is it ok to keep the component event though its not used?

Reply:
Please remove the component too